onnxruntime/core/framework/debug_node_inputs_outputs_utils.cc

```diff
@@ -175,35 +175,44 @@ const NodeDumpOptions& NodeDumpOptionsFromEnvironmentVariables() {
   static const NodeDumpOptions node_dump_options = []() {
     namespace env_vars = debug_node_inputs_outputs_env_vars;
 
+    // returns a pair with first element indicating if the env variable was set
```

Comment:
can we use this function instead? it supports a default value if GetEnvironmentVar() returns an empty string. technically that does not mean that it is unset, as it could be set to an empty string. but i digress...
https://github.com/microsoft/onnxruntime/blob/bc1768c7f13cde766180a707d132c7b047d1b28a/onnxruntime/core/platform/env_var_utils.h#L36

Reply:
sorry, what i meant was can we replace get_bool_env_var() with ParseEnvironmentVariable()? the latter should work with bools too.

Reply:
I'll leave this refactoring for a separate PR as it's not the goal of this PR.